actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -2563,11 +2562,11 @@ def file_field(method, options = {})
       # Add the submit button for the given form. When no value is given, it checks
       # if the object is a new resource or not to create the proper label:
       #
-      #   <%= form_for @post do |f| %>
+      #   <%= form_for @article do |f| %>
       #     <%= f.submit %>
       #   <% end %>
       #
-      # In the example above, if <tt>@post</tt> is a new record, it will use "Create Post" as
+      # In the example above, if <tt>@article</tt> is a new record, it will use "Create Post" as
       # submit button label; otherwise, it uses "Update Post".
```

Comment:
```suggestion
      # In the example above, if <tt>@article</tt> is a new record, it will use "Create Article" as
      # submit button label; otherwise, it uses "Update Article".
```